### PR TITLE
Support for macCatalyst and tvOS

### PR DESCRIPTION
**Pull request by hvge**
This PR contains the following changes:

- Adds support for macCatalyst platform
- Adds support for tvOS platform
- Removes deprecated `PowerAuth2-Debug` pod
- Bumped min. supported iOS to 9.0
- Fixed compatibility with Xcode12
- OpenSSL upgrade to `1.1.1h`

⚠️ Be aware that CocoaPods integration requires a  pod tool in version `1.10` and later.

### CODE REVIEW

File: PowerAuth2.podspec [hunk 1]
```diff
@@ -1,17 +1,18 @@
 Pod::Spec.new do |s|
 	# General information
+    s.cocoapods_version = '>= 1.9'
```
Comment (petrdvorak) [inline PowerAuth2.podspec:3]: Tabs vs. spaces inconsistency...

File: proj-xcode/Classes/PowerAuth2.h [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2018 Wultra s.r.o.
+ * Copyright 2018-2020 Wultra s.r.o.
```
Comment (petrdvorak) [inline proj-xcode/Classes/PowerAuth2.h:2]: Let's only keep the last year - `2020`.

File: PowerAuth2.podspec (hunk 1 above)
Comment (hvge) [reply PowerAuth2.podspec:3]: I converted all podspecs to spaces (ruby)

**Review by petrdvorak — APPROVED** [review]
Looks good 👍 

I suggest asking some of our partners to try this in a branch mode first and then merge.

**Comment by hvge** [comment]
OK, I'll keep this branch available for a while.

**Pull request merged**